recipes/extract_model/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "extract_model" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/axiom-data-science/extract_model/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5301497bb31089cebe546aaee31b1c4c241f5e1fd45f66bd814f7977dc205e65
+  # sha256 is the preferred checksum -- you can get it for a file with:
```

Comment:
Can you remove the comments from the example

Reply:
Yes will do.